src/components/backend-ai-folder-explorer.ts

```diff
@@ -1564,7 +1564,7 @@ export default class BackendAIFolderExplorer extends BackendAIPage {
       indicator,
     )
       .then(async (res) => {
-        const service_info = res.service_ports;
+        const service_info = res.service_ports ?? res.servicePorts;
```

Comment:
Please add comments indicating from which manager version the variable name has changed.